cal/cal/static/js/app.js

```diff
@@ -1,11 +1,59 @@
-var analyticsApp = angular.module('analyticsApp', ['ui.calendar']);
+var analyticsApp = angular.module('analyticsApp', ['nvd3', 'ui.calendar']);
 
 analyticsApp.controller('LoggedInCtrl', function LoggedInController($scope) {
 
   $scope.testString = "hello angular 1.5!";
 
 });
 
+// Controller to generate graph data from chart directive, cumulative tags
+analyticsApp.controller('TagsController', function($scope, $http){
+  var url = '/v1/tags';
+});
+
+// Example line graph in categories, line graph per week
+analyticsApp.controller('CategoriesController', function($scope, $http){
+  var url = '/v1/colorcategories';
```

Comment:
For consistencies' sake, "/v1/colorcategories.json". Surprised this worked, since GET requests usually return HTML. Perhaps it's smart and picks up on http headers